apollo-compiler/src/main/kotlin/com/apollographql/android/compiler/ClassNames.kt

```diff
@@ -4,6 +4,7 @@ import com.apollographql.android.api.graphql.Mutation
 import com.apollographql.android.api.graphql.Operation
 import com.apollographql.android.api.graphql.Query
 import com.apollographql.android.api.graphql.ResponseReader
+import com.apollographql.android.api.graphql.internal.Optional
```

Comment:
outside the scope of the PR, but I feel Optional shouldn't be in the `internal` package since we're exposing that API to consumers.